### ISSUE TITLE
HW4 ready for grading!

### ISSUE THREAD

**Issue by oscarsjostrand** [oscarsjostrand/su-mt4007#5]

**Comment by Tony-Fayez** [comment]
For the task Joining data:

1. It would be preffered to see exactly each step and display the table's you work with to reach the end result. 
2.  Nice plot and good description of the results presented. Everything look's good regarding this task. 

SQL

1. Every query is elegant and simple yielding the desired result. Everthing look's good.

Regex

1. Not much to add, the code is simple and easy to understand and all the result's look good.

In conclusion:
Every desired result is satisfied, till next time you could maybe add more comment's in your code so that it's easier to follow each step.

**Comment by LeoLevenius** [comment]
# $${\color{red}\text{Komplettering tarvas.}}$$

1. Missing a table for the first part.
2. Write in the text how you handle missing values and why.
3. The final plot should be top 5 countries over 10 years, not 9 countries over 5 years.

**Comment by LeoLevenius** [comment]
# $${\color{green}\text{Godkänt.}}$$
